ingestion_server/ingestion_server/qa.py

```diff
@@ -10,65 +10,122 @@ class QAScores(Enum):
     NOT_RELEVANT = 3
 
 
-def create_search_qa_index():
-    test_idx = 'search-qa'
-    _phrase_relevance(test_idx)
+def create_search_qa_index(table):
+    # This changes search-qa for images to search-qa-image
+    test_idx = f'search-qa-{table}'
+    if table == 'image':
+        _phrase_relevance_images(test_idx)
+    elif table == 'audio':
+        _phrase_relevance_audio(test_idx)
 
 
-def test_image(title, tags, creator, relevance):
+def _test_doc_attrs(title, tags, creator, relevance):
     _id = random.randint(0, 1000000000)
     sample_url = 'https://example.com/'
-    img = Image(
-        _id=_id,
-        id=_id,
-        title=title,
-        identifier=relevance,
-        creator=creator,
-        creator_url=sample_url,
-        tags=tags,
-        created_on=None,
-        url=sample_url,
-        thumbnail='',
-        provider='test',
-        source=sample_url,
-        license='by',
-        license_version='3.0',
-        foreign_landing_url=sample_url,
-        metadata=None,
-        view_count=0
-    )
+    return {
+        '_id': _id,
```

Comment:
Are both of these `id` fields necessary?

Reply:
This [doc page](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-id-field.html) made it seem like a useful field. Plus I'm generally hesitant to touch things that work 😄.

Reply:
Sorry, I added a comment to the wrong line. On line 27, there is another `id` field. I suspect that it corresponds to the now deprecated `id` field from the image_view database. If it does, then it should probably be removed because we no longer have `id` in sample data (we have `identifier` instead)